### ISSUE TITLE
Hotspot Image Connector page accessibility issues

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1397]
- Bubbles that open on focus cannot take focus and close when you try to move your mouse over them (as when mouse moves over the gap between hotspot and bubble it triggers close). Therefore, links etc. in the bubble can't be clicked.
- There's no info available to screen reader users re. where hotspots will open links (e.g. in new windows)
- No default value set in xwd for show tooltip property at project level so it never shows in the editor.
- It's possible to add audio players set to open in bubbles in the editor but this does not work in page model.
